boilerplate/_lib/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13
+
+ENV GOCILINT_VERSION="1.31.0"
```

Comment:
I like that specific versions of all these packages are supplied. Should we move them out into a config file imported by make and supply these as build args to docker? Should boilerplate have a common config file with versions of all dependencies it uses?

Reply:
It looks like prow [does not support](https://github.com/openshift/ci-tools/blob/master/CONFIGURATION.md#images) build arguments for docker images at the moment. So these versions will have to stay here for the time being.

Reply:
Just concerned about having multiple sources of truth for dependency versions.

Reply:
Once we containerize all the make targets we can remove all the other scripts/helpers which install dependencies and then the Dockerfile should be the source-of-truth.